tobiasHe/assignments/heaps/lc295/lc295.js

```diff
@@ -0,0 +1,35 @@
+// Problem Statement #
+
+// Design a class to calculate the median of a number stream. The class should have the following two methods:
```

Comment:
Good start to the overview.  Seems like you have a good understanding the conceptual approach and can work on implementing the solution in code.